lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -165,6 +166,10 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
   /// Decode (and cache) a SourceLoc.
   SILLocation::DebugLoc decodeSourceLoc(SourceLoc SL);
 
+  bool doesEmitCodeView() {
```

Comment:
Please write a getDebugInfoFormat() accessor instead. The way this function is named it implies that IRGenDebugInfo could emit both CodeView and DWARF, but the way the function is used it clearly implies that it's either or.